### PR TITLE
docs: update quickstart guide to include swift package manager command

### PR DESCRIPTION
**Pull request by adincebic**
I noticed that quickstart guide does not mention that users need to run `swift package resolve`prior to running `bazel mod tidy`.

### CODE REVIEW

**Comment by cgrindel** [comment]
@adincebic Thanks for the PR. Can you run `bazel run //:tidy` on the repo? It will update the markdown TOC and ensure everything is up-to-date.

File: README.md [hunk 1]
```diff
@@ -152,11 +153,15 @@ feel free to populate the rest of the manifest so that your package works proper
 manager. Just note that the Swift Gazelle plugin does not use the manifest to generate Bazel build
 files, at this time.
 
-### 4. Run `bazel mod tidy`.
+### 4. Run `swift package resolve`
```
Comment (cgrindel) [inline README.md:156]: I think that we want to recommend `swift package update`. The `resolve` command will always try to find the latest version of the transitive deps. That is what the `swift_update_packages` macro would do by default.

Comment (adincebic) [reply README.md]: Done.

**Comment by adincebic** [comment]
> @adincebic Thanks for the PR. Can you run `bazel run //:tidy` on the repo? It will update the markdown TOC and ensure everything is up-to-date.

Done.

**Review by cgrindel — APPROVED** [review]

**Pull request merged**